swift/Tests/CoinAddressDerivationTests.swift

```diff
@@ -142,6 +142,9 @@ class CoinAddressDerivationTests: XCTestCase {
                 case .qtum:
                     let expectedResult = "QhceuaTdeCZtcxmVc6yyEDEJ7Riu5gWFoF"
                     AssetCoinDerivation(coin, expectedResult, derivedAddress, address)
+                case .nuls:
+                    let expectedResult = "NsdtNvsfmPerWk4BhcapHTB3LptF8Sbe"
+                    AssetCoinDerivation(coin, expectedResult, derivedAddress, address)
```

Comment:
Any other HD wallet support NULS? could you please make sure same recovery phrase generate same address?

Reply:
I remove this case code.

Reply:
why remove? just make sure we can be compatible with other wallets